app/code/Magento/CatalogMessageBroker/Model/FetchProductVariants.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogMessageBroker\Model;
+
+use Magento\CatalogMessageBroker\HttpClient\RestClient;
+use Magento\CatalogExport\Event\Data\Entity;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @inheritdoc
```

Comment:
@inheritdoc should be used only on class members

Reply:
Resolved